pkg/logs/input/docker/convertor.go

```diff
@@ -0,0 +1,160 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+// +build docker
+
+package docker
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+	iParser "github.com/DataDog/datadog-agent/pkg/logs/parser"
+	"regexp"
+)
+
+var (
+	delimiter = []byte{' '}
+	// 2019-05-29T09:26:32.155255473Z
+	timestampMatcher = regexp.MustCompile("\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{9}Z")
+	// Escaped CRLF, used for determine empty messages
+	windowsEOFMark     = []byte{'\\', 'r', '\\', 'n'}
+	carriageReturnMark = []byte{'\\', 'r'}
+	lineFeedMark       = []byte{'\\', 'n'}
+)
+
+const (
```

Comment:
Could you group constants that are related, i.e:
```go
// message indexes
indexOfHeader      = 0
indexOfContent     = 1
// header indexes
indexOfStatus      = 0
indexOfContentSize = 4
indexOfTimestamp   = 8
```

Reply:
will do